include/sys/fs/zfs.h

```diff
@@ -1134,6 +1134,11 @@ typedef enum zfs_ioc {
  */
 #define	BLKZNAME		_IOR(0x12, 125, char[ZFS_MAX_DATASET_NAME_LEN])
 
+/*
+ * zpl ioctl to get number of filled blocks
+ */
+#define	ZFS_IOC_COUNT_FILLED	_IOR('f', 100, uint64_t)
```

Comment:
Rather than using the global `'f'` magic here, each filesystem is allowed to use their own unique magic for custom ioctl()'s to prevent collisions with the generic ones.  This will be our first for ZFS so we'll need to pick a magic value, how about 'Z' which looks to be unused.  We should also then be able to start at 1.

```
#define ZFS_IOCTL_MAGIC 'Z'

#define ZFS_IOC_COUNT_FILLED	_IOR(ZFS_IOCTL_MAGIC, 1, uint64_t)
```

Reply:
According to `linux/Documentation/ioctl/ioctl-number.txt` the `Z` space is in (limited) use:

    'Z'     14-15   drivers/message/fusion/mptctl.h

I'd suggest to start at 32 or 64 to avoid potential conflicts.

Reply:
I changed it to 'Z' but kept it at number 100.